package.json

```diff
@@ -30,8 +30,8 @@
   "private": true,
   "scripts": {
     "boot": "lerna bootstrap --reject-cycles",
-    "clear": "lerna run clear",
-    "dist": "yarn clear && lerna run dist",
+    "clean": "lerna run clean",
+    "dist": "yarn clean && lerna run dist",
```

Comment:
I think yarn does not have `clean` command anymore, you may want to update to use `autoclean`